examples/ts/apt/fund-forwarders.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
This file shoudn't need a WP review. Please update the CODEOWNERS file accordingly

Reply:
I've mapped the `examples` directory to `BitGo/ethalt-team`.